src/cloudant/account.py

```diff
@@ -221,14 +201,14 @@ def delete_database(self, dbname):
 
     def db_updates(self, since=None, continuous=True):
         """
-        _db_updates_
-
-        Implement streaming from _db_updates feed. Yields information about
-          databases that have been updated.
+        Streams data from _db_updates feed. Yields information about
+        databases that have been updated.
 
-        :param str since: Start from this sequence
-        :param boolean continuous: Stream results
+        :param str since: Starts update streaming from this sequence
```

Comment:
Some parameter comments end with periods and this is one that doesn't.  I do see other comments below with no periods that are incomplete sentences.  Would it be best to have them all complete sentences with periods?